### PR TITLE
feature/ztna-remove

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Remove ZTNA

### CODE REVIEW

File: conf/caddy-services/api.conf.example [hunk 1]
```diff
@@ -2,21 +2,11 @@
...
-    except /api/v1/remote_clients/events
-    except /api/v1/remote_clients/my_events
-  }
-
   errors {
     502 /usr/local/pf/html/pfappserver/root/errors/502.json
     503 /usr/local/pf/html/pfappserver/root/errors/503.json
   }
 
-  pfconfigpool {
```
Comment (julsemaan) [inline conf/caddy-services/api.conf.example:15]: You can't remove this whole section, the pfconfigpool directive needs to stay

File: conf/caddy-services/api.conf.example [hunk 2]
```diff
@@ -2,21 +2,11 @@
 0.0.0.0:9999 {
   tls /usr/local/pf/conf/ssl/server.crt /usr/local/pf/conf/ssl/server.key
 
-  log / stdout "api-frontend-access {remote} - {user} [{when}] \"{method} {uri} {proto}\" {status} {size} \"{>Referer}\" \"{>User-Agent}\"" {
```
Comment (julsemaan) [inline conf/caddy-services/api.conf.example:5]: You need to leave the log directive

**Comment by extrafu** [comment]
@julsemaan and @fdurand are we ready to merge? We need to get this in ASAP to make sure it doesn't break testing.

**Comment by julsemaan** [comment]
> @julsemaan and @fdurand are we ready to merge? We need to get this in ASAP to make sure it doesn't break testing.

I put comments that still need @jrouzierinverse's attention since they introduce regressions

**Comment by jrouzierinverse** [comment]
That was already taken care of.

**Comment by fdurand** [comment]
On my side i am ok, looks that the dns portion has been removed correctly.

**Comment by julsemaan** [comment]
I'm OK too

**Pull request merged**